feature/steps/login_steps.py

```diff
@@ -24,4 +24,4 @@ def user_is_redirected_to_home_page(context):
     wait_until_text_is_present(context,'#logout2', 'Log out')
```

Comment:
remember to move this selectors to the page.

also I'm looking at this step and it says `the user is redirected to the home of the application`
remember that you are not redirected to the home when you log in.
those refactors need to be done, create a branch to fix those tests if you don't want to do them on this branch.